tensorboard/plugins/text/summary_v2.py

```diff
@@ -26,10 +26,44 @@
 def text(name, data, step=None, description=None):
     """Write a text summary.
 
+    See also `tf.summary.scalar`, `tf.summary.SummaryWriter`, `tf.summary.image`.
+
+    Logs text Tensor values for later visualization and analysis in TensorBoard.  Like
```

Comment:
Same comment as other PR re: mentioning default writer, consistency w.r.t. choice of leading verb.